compiler/src/iree/compiler/Dialect/Flow/Transforms/test/annotate_dispatches.mlir

```diff
@@ -266,3 +266,80 @@ flow.executable private @ex {
     }
   }
 }
+
+// -----
+
+#map = affine_map<(d0, d1, d2) -> (d0, d2)>
+#map1 = affine_map<(d0, d1, d2) -> (d2, d1)>
+#map2 = affine_map<(d0, d1, d2) -> (d0, d1)>
+
+flow.executable private @ex {
+  // CHECK: flow.executable.export public @dispatch_contract_16x32x8_f32
```

Comment:
Can you clarify for this test case what part of the code is "contract"? Maybe the affine maps satisfy `isaContractionOpInterface`? Similar comment for conv and elementwise

Reply:
Yeah. I'll have follow up patches to improve the "contract" category later--it's useful to know that if underlying it's matmul.